src/renderer/components/title-bar/title-bar.component.tsx

```diff
@@ -89,23 +88,24 @@ export default function TitleBar({ template = "index.html" }: { template: AppWin
                     </div>
                     <div onClick={minimizeWindow} className="text-gray-800 dark:text-gray-200 hover:bg-gray-300 dark:hover:bg-[#4F545C] cursor-pointer w-11 h-full shrink-0 flex justify-center items-center" id="min-button">
                         <svg aria-hidden="false" width="12" height="12" viewBox="0 0 12 12">
-                            <rect fill="currentColor" width="10" height="1" x="1" y="6">
-                                {" "}
-                            </rect>
+                            <rect fill="currentColor" width="10" height="1" x="1" y="6" />
                         </svg>
                     </div>
-                    <div onClick={toogleMaximize} className="text-gray-800 dark:text-gray-200 hover:bg-gray-300 dark:hover:bg-[#4F545C] cursor-pointer w-11 h-full shrink-0 flex justify-center items-center" id="max-button">
+                    <div onClick={toggleMaximize} className="text-gray-800 dark:text-gray-200 hover:bg-gray-300 dark:hover:bg-[#4F545C] cursor-pointer w-11 h-full shrink-0 flex justify-center items-center" id="max-button">
                         <svg aria-hidden="false" width="12" height="12" viewBox="0 0 12 12">
-                            <rect width="9" height="9" x="1.5" y="1.5" fill="none" stroke="currentColor">
-                                {" "}
-                            </rect>
+                            {maximized ? (
+                                <>
+                                    <rect width="9" height="9" x="0.5" y="2.5" fill="none" stroke="currentColor" />
+                                    <path d="M 2.5 2.5 V 0.5 H 11.5 V 9.5 H 9.5" fill="none" stroke="currentColor" />
+                                </>
+                            ) : (
+                                <rect width="9" height="9" x="1.5" y="1.5" fill="none" stroke="currentColor" />
+                            )}
                         </svg>
                     </div>
```

Comment:
```suggestion
                    </button>
```

do that for all div using "onClick" and it will be good 👍